selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/ecms/ECMSLocator.java

```diff
@@ -1147,4 +1147,11 @@
   public static final SelenideElement ELEMENT_LIST_FOLDER_IN_DEFAULT_VIEW=$(byId("UIDocumentWorkspace"));
 
   public static final SelenideElement ELEMENT_BUTTON_CONFIRM_DELETE_FILE=$(byClassName("UIDeleteFileConfirmMessage")).find(byClassName("OK"));
+  public static final SelenideElement ELEMENT_SEARCH_BTN=$(byId("simpleSearch"));
+  public static final SelenideElement ELEMENT_ICON_SEARCH=$(byClassName("uiIconSearch"));
+  public static final SelenideElement ELEMENT_TITLE_WEB_CONTENT=$(byClassName("title"));
+
+
+
+
```

Comment:
Use same formatting than others variables and remove useless blank lines please.

Reply:
done